Android/app/src/main/java/realm/io/realmpop/controller/PlayerRecyclerViewAdapter.java

```diff
@@ -33,6 +34,11 @@ public void onBindViewHolder(final ViewHolder holder, final int position) {
         final Player player = getData().get(position);
         holder.playerId = player.getId();
         holder.titleView.setText(player.getName());
+        if (player.isAvailable()) {
+            holder.titleView.setTextColor(Color.parseColor("#FCC397"));
```

Comment:
I tried, but it always loaded a different color than the name I've been using, it was weird and I gave up :)